### PR TITLE
Add blogging guide, communications plan, index tidying 

### LINKED ISSUE #131: Publish communications plan on About
> Comms plan lite: how can people can keep up with our news? Define use of channels so we can tell people where to look on Thurs.
> 
> + Twitter
> + FB page
> + mailing list for people to sign up to?
> + blog?
> + Planable
> 
> 19 April 2019: [Unfinished draft here](https://docs.google.com/document/d/1vG6L2xHrGbTqLEwNOrRHFeub3geFd-k_1qzHJY4eJ6o/edit?usp=sharing)
> 
> Agree what we communicate about, how often, and who does it.
> 
> @nebster @elenafdr @bvhme

### PR DESCRIPTION
**Pull request by ElenaFdR**
Draft pending a second opinion on the index arrangement.

Closes #131.

-----
[View rendered activities/communication/blogging.md](https://github.com/publiccodenet/about/blob/blogging-and-comms/activities/communication/blogging.md)
[View rendered activities/communication/email-signature-template.md](https://github.com/publiccodenet/about/blob/blogging-and-comms/activities/communication/email-signature-template.md)
[View rendered activities/communication/index.md](https://github.com/publiccodenet/about/blob/blogging-and-comms/activities/communication/index.md)
[View rendered activities/communication/managing-voip-phone-numbers.md](https://github.com/publiccodenet/about/blob/blogging-and-comms/activities/communication/managing-voip-phone-numbers.md)

### CODE REVIEW

**Comment by ElenaFdR** [comment]
Dependency: publishing the comms plan, or linking to the Google doc.

File: activities/communication/blogging.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+1\. Since none of us work work in isolation, check in with the communications coordinator before you start writing your blogpost. This helps us:
+
+* understand your goals and audience for the blogpost
+* establish the scope of your blogpost (to avoid duplication with other planned work)
+* schedule publication of your blogpost so it fits best with other planned communications
+
+2\. [Read our style guide](https://content-guide.18f.gov/) - we use 18F's. If you have any questions, ask!
+
+3\. Make sure your information is correct on [blog/_data/staff.csv](https://github.com/publiccodenet/blog/blob/develop/_data/staff.csv).
```
Comment (bvhme) [inline activities/communication/blogging.md]: This is a list with a nested sublist, please use normal MarkDown syntax for that.

File: activities/communication/blogging.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+## Write your blogpost
+
+Write your draft blogpost on HackMD.io. This lets you share it for pair writing, factchecking, feedback, or proofreading.
```
Comment (bvhme) [inline activities/communication/blogging.md]: I'd like to argue against choosing a specific tool in this guide as I think it should be up to those doing the work to choose their own toolset.

File: activities/communication/blogging.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+Read the [guide on writing community call blogposts](community-call-blogposts.md)
+
+After writing, please share with the comms coordinator for polishing.
+
+## How to create a blogpost on the Foundation’s blog
+
+1\. Create a branch for your new blogpost within in the [blog repo](https://github.com/publiccodenet/blog).
+
+2\. Create a new file for your blogpost within blog/BLOG/_posts/. Name your file  ```YEAR-MONTH-DAY-title.md```.
```
Comment (bvhme) [inline activities/communication/blogging.md]: What is this path, I see no `blog/BLOG/_posts` in any repository. I do see a `_posts` folder in the `blog` repository.

Code inline only needs one backtick. The three backticks are to indicate a block is code. Additionally the path here needs to be code as well.

```suggestion
2\. Create a new file for your blogpost within [`_posts/` on the blog repository](https://github.com/publiccodenet/blog/tree/develop/_posts). Name your file  `YEAR-MONTH-DAY-title.md`.
```

**Comment by ElenaFdR** [comment]
@bvhme @Ainali, this is finally not a draft! I've added the communications plan, and reorganized and tightened this all a bit further.

File: activities/communication/blogging.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+1. Since none of us work work in isolation, check in with the communications coordinator before you start writing your blogpost. This helps us:
+
+* understand your goals and audience for the blogpost
+* establish the scope of your blogpost (to avoid duplication with other planned work)
+* schedule publication of your blogpost so it fits best with other planned communications
+
+2. [Read our style guide](https://content-guide.18f.gov/) - we use 18F's. If you have any questions, ask!
+
+3. Make sure your information is correct on [blog/_data/staff.csv](https://github.com/publiccodenet/blog/blob/develop/_data/staff.csv).
```
Comment (bvhme) [inline activities/communication/blogging.md]: If this is a list, with a sublist, it should not have the newlines. If it is not a list it should not have the numbers.

File: activities/communication/blogging.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+```
+![alternative text](/assets/image.jpg)
+```
+
+[Make the text alternative for your image useful](https://www.w3.org/WAI/tips/writing/#write-meaningful-text-alternatives-for-images).
+
+All blogposts should include at least one relevant image, to make them more engaging.
+
+5. Approvals: tag the communications coordinator on the pull request for your blogpost for a final proofread.
```
Comment (bvhme) [inline activities/communication/blogging.md]: This can be also without the numbers or alternatively with headings for each step. The text already has an order.

File: activities/communication/communications-plan.md [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+3. Build our codebase community and attract more contributors
+4. Get more contributors and funding for the Foundation
+5. Boost the public code ecosystem
+
+## Audience
+
+Our audience is divided into:
+
+* current Foundation users
+* future Foundation users
```
Comment (bvhme) [inline activities/communication/communications-plan.md]: Even though it has some content below, I think 'user' really needs some sort of definition here, as it feels quite out of place.

File: activities/communication/communications-plan.md [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+* influencers
+
+### Foundation users
+
+We believe [these are the users of the Foundation for Public Code](https://about.publiccode.net/activities/value-and-impact/user-mapping/).
+
```
Comment (bvhme) [inline activities/communication/communications-plan.md]:
```suggestion
```

**Comment by bvhme** [comment]
Also, I wanted to add this is great and monumental, albeit a bit of a big chunk to push in all at once.

File: activities/communication/community-call-blogposts.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+* [how to run a community call](run-a-community-call.md)
+* [how to write a blogpost](blogging.md)
+
+## At the top of the page
+
+Make sure you:
+
+* format your title like previous community call blogposts
+* make sure your title is the same in both the H1 and the frontmatter
```
[the remark this answers is not in the document]
Comment (Ainali) [reply activities/communication/community-call-blogposts.md]: Oh, if that is the case it is not needed. Then we collapse those two bullet points into one "* format your H1 like previous community call blogposts".

[the remark this answers is not in the document]
Comment (Ainali) [reply activities/communication/community-call-blogposts.md]: Now we remembered, it's picked up on the page, but not by the functionality that renders the summary on the front page. So unfortunately we need to add it in both the frontmatter and H1.

File: activities/communication/communications-plan.md [hunk 9]
```diff
@@ -0,0 +1,189 @@
...
+* [political](../value-and-impact/user-mapping/political.html)
+* [senior civil servants](../value-and-impact/user-mapping/senior-civil-servants.html)
+* [structural funding and philanthropy](../value-and-impact/user-mapping/structural-funding-philanthropy.html)
+
+Messages:
+
+* Find out more about [place or institution’s] interesting public code(ish) approach/project
+* Find out more about the Foundation’s work (new member signed up, new codebase incubated)
+* Find out more about the Foundation’s vision
```
Comment (bvhme) [inline activities/communication/communications-plan.md]:
```suggestion
* Find out more about the Foundation for Public Code its work (new member signed up, new codebase incubated)
* Find out more about the Foundation for Public Code its vision
```

File: activities/communication/communications-plan.md [hunk 10]
```diff
@@ -0,0 +1,189 @@
...
+* Potentially: announcing when a new vendor becomes available to support a specific codebase (like [this OS2 example](https://os2.eu/blog/en-ny-leverandoer-til-os2kitos-fra-juni-2019))
+
+## Channels
+
+Our channels are:
+
+* [Our blog]: long form, platform of record (read more about our blog)
+* [Twitter]: short form, ephemera
+* [Facebook]: medium form, ephemera (mostly avoid, since it’s a closed platform)
+* [LinkedIn]: reshare content created about us and posted by others, or replicate FB (mostly avoid, since it’s a closed platform)
```
Comment (bvhme) [inline activities/communication/communications-plan.md]: These should be links I'm guessing. Except for Facebook and LinkedIn that should just be words.

**Comment by bvhme** [comment]
Hey @ElenaFdR, are we still persuing closing this pull request or should it be closed?

File: activities/communication/blogging.md [hunk 11]
```diff
@@ -0,0 +1,91 @@
...
+## Write your blogpost
+
+Write your draft blogpost using a collaborative document tool. This lets you share it for pair writing, factchecking, feedback or proofreading.
+
+Don't hesitate to involve other people in your writing process - we want you to succeed, and we know how helpful an external perspective can be once something starts to feel like a problem.
+
+After writing, please share with the communications coordinator for polishing.
+
+## How to create a blogpost on the Foundation’s blog
+
```
[the remark this answers is not in the document]
Comment (ElenaFdR) [reply activities/communication/blogging.md:54]: Hey, I've tried 3 different approaches to indenting the codeblocks, with no joy. I'd welcome any suggestions for what else to try. So far I've tried 4 spaces, 2 spaces and 1 space (based on [this and its commments](https://gist.github.com/clintel/1155906).

[the remark this answers is not in the document]
Comment (bvhme) [reply activities/communication/blogging.md:54]: I'll make a commit structuring this.

[the remark this answers is not in the document]
Comment (bvhme) [reply activities/communication/blogging.md:54]: The main issue being that lists are not meant for containing such complex content and don't support including fenced code blocks.

[the remark this answers is not in the document]
Comment (bvhme) [reply activities/communication/blogging.md:54]: I've made #645 as a suggestion.

File: activities/communication/communications-plan.md [hunk 12]
```diff
@@ -0,0 +1,193 @@
...
+
+To enable public-purpose software and policy that is open and collaborative.
+
+## Business objectives
+
+1. Get more members ([in phase 1](https://about.publiccode.net/organization/roadmap.html))
+2. Boost public codebases under Foundation stewardship
+3. Build our codebase community and attract more contributors
+4. Get more contributors and funding for the Foundation
+5. Boost the public code ecosystem
```
[the remark this answers is not in the document]
Comment (ElenaFdR) [reply activities/communication/communications-plan.md:32]: I've replaced this with a link to the soon-to-be-published priorities.

File: activities/communication/communications-plan.md [hunk 13]
```diff
@@ -0,0 +1,193 @@
...
+## Metrics
+
+These are statistics we can easily track month on month as a proxy for impact:
+
+* tweet numbers
+* new Twitter followers
+* Twitter profile visits
+* Twitter mentions
+* tweet impressions
+* increased use of #publiccode
```
[the remark this answers is not in the document]
Comment (ElenaFdR) [reply activities/communication/communications-plan.md:180]: I've removed the whole metrics section.

File: activities/communication/communications-plan.md [hunk 14]
```diff
@@ -0,0 +1,193 @@
...
+
+- Announcement that thing is happening (give teaser)
+- Join in on other related conversations and hashtags (monitor conversation)
+- Tweet about the thing as it’s happening
+- Publish blog during (if a major announcement/unveiling) or afterwards about what happened, plus what we learned or what’s next
+- Measure how this worked
+
+## Metrics
+
+These are statistics we can easily track month on month as a proxy for impact:
```
[the remark this answers is not in the document]
Comment (ElenaFdR) [reply activities/communication/communications-plan.md:173]: I've removed the whole metrics section, and instead created #646.

File: activities/communication/communications-plan.md [hunk 15]
```diff
@@ -0,0 +1,169 @@
...
+* active, dynamic, getting things done
+* smart enough to be trusted with the importance and complexity of its work
+* plugged into and supportive of the network of visionaries and activists who also want this future
+* open and transparent, both about its intentions and operations as well as learnings, failures and successes
+
+We believe it’s more important to talk concretely about what we’ve already done than what we hope to do.
+
+## Objectives
+
+Our communications should help us achieve [our current priorities](../../organization/mission.md#priorities).
```
Comment (bvhme) [inline activities/communication/communications-plan.md:28]: The heading is named 'Current priorities'

```suggestion
Our communications should help us achieve [our current priorities](../../organization/mission.md#current-priorities).
```

File: activities/communication/communications-plan.md (hunk 6 above)
Comment (ElenaFdR) [reply activities/communication/communications-plan.md:35]: This use is defo out of sync with #603.

**Comment by ElenaFdR** [comment]
@bvhme Take a look now? I think I've addressed all your comments.

**Comment by ElenaFdR** [comment]
Bah - my last (very creative!) commit broke the build.

File: activities/communication/blogging.md [hunk 16]
```diff
@@ -0,0 +1,98 @@
...
+1. Since none of us work in isolation, check in with the communications coordinator before you start writing your blogpost. This helps us:
+  * understand your goals and audience for the blogpost
+  * establish the scope of your blogpost (to avoid duplication with other planned work)
+  * schedule publication of your blogpost so it fits best with other planned communications
+
+2. [Read our style guide](https://content-guide.18f.gov/) - we use 18F's. If you have any questions, ask!
+3. Make sure your information is correct on [blog/_data/staff.csv](https://github.com/publiccodenet/blog/blob/develop/_data/staff.csv).
+4. Read the [guide on writing community call blogposts](community-call-blogposts.md) if that's what you're doing.
+
```
Comment (bvhme) [inline activities/communication/blogging.md:41]: This is 1 list with a sublist, not 2 lists of which one has a sublist.

Whitespace (line 37) removed:

```suggestion
1. Since none of us work in isolation, check in with the communications coordinator before you start writing your blogpost. This helps us:
  * understand your goals and audience for the blogpost
  * establish the scope of your blogpost (to avoid duplication with other planned work)
  * schedule publication of your blogpost so it fits best with other planned communications
2. [Read our style guide](https://content-guide.18f.gov/) - we use 18F's. If you have any questions, ask!
3. Make sure your information is correct on [blog/_data/staff.csv](https://github.com/publiccodenet/blog/blob/develop/_data/staff.csv).
4. Read the [guide on writing community call blogposts](community-call-blogposts.md) if that's what you're doing.

```

File: activities/communication/communications-plan.md [hunk 17]
```diff
@@ -0,0 +1,169 @@
...
+
+### Influencers
+
+These are domain celebrities in open source, government digital transformation and civic tech.
+
+## Unique selling proposition
+
+We are the only organization focused on open source software for public organizations that:
+
+* is non-profit and member owned (unlike commercial open source govtech companies)
```
Comment (bvhme) [inline activities/communication/communications-plan.md:50]: This needs a qualifier 'with membership only open to public organizations' if this needs to be true on it's own.

File: activities/communication/communications-plan.md [hunk 18]
```diff
@@ -0,0 +1,169 @@
...
+* offers codebase stewardship for different projects (how are we different from the Commons Conservancy and the businesses they can put you in touch with?)
```
Comment (bvhme) [inline activities/communication/communications-plan.md:51]: I don't completely know what this means.

File: activities/communication/communications-plan.md [hunk 19]
```diff
@@ -0,0 +1,169 @@
...
+
+We believe [these are potential contributors to the Foundation for Public Code](https://about.publiccode.net/activities/value-and-impact/user-mapping/).
+
+### Influencers
+
+These are domain celebrities in open source, government digital transformation and civic tech.
+
+## Unique selling proposition
+
+We are the only organization focused on open source software for public organizations that:
```
Comment (bvhme) [inline activities/communication/communications-plan.md:48]: This list would make more sense if it was clear that not any of these but all of these together make us what we are, as all of them are done by others.

File: activities/communication/running-community-call.md [hunk 20]
```diff
@@ -102,4 +102,4 @@ The notetaker reworks the call notes into a call summary.
 The notetaker shares the call summary with people who were on the call for amendment and approval before:
 
 * linking from any relevant GitHub issues or pull requests
-* publishing on [the blog](https://blog.publiccode.net)
```
Comment (bvhme) [inline activities/communication/running-community-call.md:105]: Should this not link to the `community-call-blogposts.md`?

File: activities/communication/blogging.md (hunk 16 above)
Comment (AlbaRoza) [inline activities/communication/blogging.md:41]: 5. Read the [Code of Conduct](CODE_OF_CONDUCT.md).

File: activities/communication/blogging.md [hunk 21]
```diff
@@ -0,0 +1,98 @@
...
+The blog is our primary tool for talking about:
+
+* our work: what we’re working on and what we’ve learned
+* our news: what we’ve achieved and what we’ll be doing next
+* our community: interviews with and guest posts by community members
+* our position on important issues: why we’ve made the political, technological, organizational or design decisions we have
+
+Not all information published on the blog needs to support our marketing strategy - for example, an in-depth discussion about a technology choice we’ve made may not persuade potential Foundation members, but will be interesting to technologists in our community.
+
+To make sure we have the editorial freedom to publish about all of our work (no matter how niche), there’s no direct link to the blog on [publiccode.net](https://publiccode.net/). Instead, we may feature important blogposts. (We’ll build this functionality once we have important blogposts.)
```
Comment (AlbaRoza) [inline activities/communication/blogging.md:21]: We can also state clearly that opinions of authors are independent from the Foundation for Public Code. Also, if blog was linked from the website, search engines would identify it easier as our main Communications tool.

File: activities/communication/blogging.md [hunk 22]
```diff
@@ -0,0 +1,98 @@
...
+4. Read the [guide on writing community call blogposts](community-call-blogposts.md) if that's what you're doing.
+
+## Write your blogpost
+
+Write your draft blogpost using a collaborative document tool. This lets you share it for pair writing, factchecking, feedback or proofreading.
+
+Don't hesitate to involve other people in your writing process - we want you to succeed, and we know how helpful an external perspective can be once something starts to feel like a slog.
+
+After writing, please share with the communications coordinator for polishing.
+
```
Comment (AlbaRoza) [inline activities/communication/blogging.md:49]: Include if possible links to people, associations, organizations or events you might be talking about.
When creating the hyperlink, configure it to open in a new tab, not in the reader's one. By doing so, we keep the reader's attention in our blog.

File: activities/communication/blogging.md [hunk 23]
```diff
@@ -0,0 +1,98 @@
...
+title:  "Title”
+date: YYYY-MM-DD
+author: [put your name]
+type: blogpost
+category: [our current categories are: news, community call and codebase stewardship]
+excerpt: [unformatted text you want to appear on the index]
+---
+```
+
+All blogposts should include at least one relevant image, to make them more engaging.
```
Comment (AlbaRoza) [inline activities/communication/blogging.md:74]: All blogposts should include at least one relevant image or video to make them more engaging.

File: activities/communication/index.md [hunk 24]
```diff
@@ -7,39 +7,23 @@ type: Index
...
 * [Using chat.publiccode.net](using-chat.md)
-* [Community discussions](community-discussions.md)
-* [Instructions on how to run a community call](running-community-call.md)
-* [Phone numbers](phone-numbers.md)
-
-Community calls are a way to get direct input from the community on topics that in any way are of concern to them. We do them on a bi-weekly schedule for regularity and they are open for anyone to join.
-
-## Using and managing phone numbers
+* [How and why to write a blogpost](blogging.md)
+* [Email signature template](email-signature-template.md)
```
Comment (AlbaRoza) [inline activities/communication/index.md:14]: I'd also add other activities related to Communications such as attending events, wether as an organizer, speaker or as an attendee, Social Media, webinars or workshops we might offer for free

File: activities/communication/communications-plan.md [hunk 25]
```diff
@@ -0,0 +1,169 @@
...
+
+## Principles
+
+Our communications need to reflect our core values: open, quality, trust, community, helpful. They must also be non-partisan, in line with our [founding principles](../../organization/mission.md#founding-principles).
+
+Our messages range from neutral in tone to positive and inspirational. Our community won’t be sustainable if we attract people motivated by fear or anger.
+
+Our communications are in plain English, but we link to highly relevant pages in other languages, for example about our members’ work.
+
+Our communications should show to showcase the Foundation for Public Code as:
```
Comment (lscheske) [inline activities/communication/communications-plan.md:17]:
```suggestion
Our communications should showcase the Foundation for Public Code as:
```

File: activities/communication/communications-plan.md [hunk 26]
```diff
@@ -0,0 +1,169 @@
...
+
+Our communications should help us achieve [our current priorities](../../organization/mission.md#current-priorities).
+
+## Audience
+
+Our audience is divided into:
+
+* current Foundation contributors
+* future Foundation contributors
+* influencers
```
Comment (lscheske) [inline activities/communication/communications-plan.md:36]: I find this very focussed on "us". Is our audience not also (people working in/for) public organizations on public code transformation, or OSS? How can we leverage the broader public code community to achieve our current priorities? This doesn't all have to be answered in this PR, but I think something to think about.

File: activities/communication/communications-plan.md [hunk 27]
```diff
@@ -0,0 +1,169 @@
...
+
+### Foundation contributors
+
+We believe [these are potential contributors to the Foundation for Public Code](https://about.publiccode.net/activities/value-and-impact/user-mapping/).
+
+### Influencers
+
+These are domain celebrities in open source, government digital transformation and civic tech.
```
Comment (lscheske) [inline activities/communication/communications-plan.md:44]: As in individual humans, or whole organizations like FSFE?

File: activities/communication/communications-plan.md (hunk 18 above)
Comment (lscheske) [reply activities/communication/communications-plan.md:51]: I understand this as us offering codebase stewardship to a number of public code projects, unlike existing public code foundations taht focus on one codebase e.g. Decidim, Consul foundations. 

I think the sentence in the brackets is confusing.

File: activities/communication/communications-plan.md [hunk 28]
```diff
@@ -0,0 +1,169 @@
...
+* offers codebase stewardship for different projects (how are we different from the Commons Conservancy and the businesses they can put you in touch with?)
+* offers productization and marketing of the codebases (unlike Commons Conservancy)
+* actually provides guidance and support to help public orgs make their code open (unlike Free Software Foundation)
+* isn’t tied to the EU project lifecycle (unlike SCORE)
+* has a reproducible process suitable not just for cities, but all public organizations (unlike SCORE)
+* is international (unlike OS2, Cloud 4 Cities, VNG-Realisatie, Adullact)
+
+We are also actively building our ecosystem by:
+
+* connecting members with like-minded parties (potential members, partners, affiliates, structural funders)
```
Comment (lscheske) [inline activities/communication/communications-plan.md:60]:
```suggestion
* connecting members with like-minded parties (other members, affiliates, public organizations, structural funders)
```

File: activities/communication/communications-plan.md [hunk 29]
```diff
@@ -0,0 +1,169 @@
...
+
+* [political](../value-and-impact/user-mapping/political.html)
+* [senior civil servants](../value-and-impact/user-mapping/senior-civil-servants.html)
+* [structural funding and philanthropy](../value-and-impact/user-mapping/structural-funding-philanthropy.html)
+
+Messages:
+
+* Find out more about [place or institution’s] interesting public code(ish) approach/project
+* Find out more about the Foundation for Public Code and its work (new member signed up, new codebase incubated)
+* Find out more about the Foundation for Public Code its vision
```
Comment (lscheske) [inline activities/communication/communications-plan.md:80]:
```suggestion
* Find out more about the Foundation for Public Code and its vision
```

File: activities/communication/communications-plan.md [hunk 30]
```diff
@@ -0,0 +1,169 @@
...
+
+We are also actively building our ecosystem by:
+
+* connecting members with like-minded parties (potential members, partners, affiliates, structural funders)
+* boosting awareness of public code
+* setting standards and creating reusable resources (by providing thought leadership!)
+* demonstrating the value of public code with well documented, public business cases and risk analyses
+* nurturing our community online, and – where sensible – in person
+
+## Calls to action
```
Comment (lscheske) [inline activities/communication/communications-plan.md:66]: Not sure whether the content that follows are calls to action. When I see "call to action", I expect some kind of arguments that will make me want to do something e.g. Join us in the fight against climate change! 

When I read the content that follows below, I see different types of information that we communicate about to different groups/stakeholders. Could we come up with a heading that's more fitting?

File: activities/communication/communications-plan.md [hunk 31]
```diff
@@ -0,0 +1,169 @@
...
+* State of the market info about/for vendors (focusing on a class of vendors, rather than endorsing specific vendors)
+* Potentially: announcing when a new vendor becomes available to support a specific codebase (like [this OS2 example](https://os2.eu/blog/en-ny-leverandoer-til-os2kitos-fra-juni-2019))
+
+## Channels
+
+Our channels are:
+
+* [Our blog](https://blog.publiccode.net/): long form, platform of record (read more about our blog)
+* [Twitter](https://twitter.com/publiccodenet/): short form, ephemera
+* LinkedIn: share our blogposts or reshare others' content about us, ephemera (mostly avoid, since it’s a closed platform)
```
Comment (lscheske) [inline activities/communication/communications-plan.md:123]:
```suggestion
* [LinkedIn](https://www.linkedin.com/company/publiccodenet/): share our blogposts or re-share others' content about us, ephemera (mostly avoid, since it’s a closed platform)
```

File: activities/communication/communications-plan.md [hunk 32]
```diff
@@ -0,0 +1,169 @@
...
+* Facebook: medium form, ephemera (mostly avoid, since it’s a closed platform)
+
+Our posting frequency is:
+
+* Twitter - as often as we have something to say, up to daily. It’s dynamic, responsive, topical
```
Comment (lscheske) [inline activities/communication/communications-plan.md:128]:
```suggestion
* Twitter: as often as we have something to say, up to daily. It’s dynamic, responsive, topical
```

File: activities/communication/communications-plan.md [hunk 33]
```diff
@@ -0,0 +1,169 @@
...
+Messages:
+
+* Good news from our partners and affiliates (eg, relevant session at organization X’s annual conference!)
+* Any news about Foundation or codebase impact for end users (eg, check out these communications by a member about their new software for managing parking)
+* State of the market info about/for vendors (focusing on a class of vendors, rather than endorsing specific vendors)
+* Potentially: announcing when a new vendor becomes available to support a specific codebase (like [this OS2 example](https://os2.eu/blog/en-ny-leverandoer-til-os2kitos-fra-juni-2019))
+
+## Channels
+
+Our channels are:
```
Comment (lscheske) [inline activities/communication/communications-plan.md:119]: Should we consider adding events as a channel? They are mentioned later on as things we do campaigns on, but are not explained anywhere here.

This would make our channels of communication more diverse, and not just restricted to social media platforms + blog. 

A thought for later: when we (re)establish our press contacts, would these also be worth mentioning here?

File: activities/communication/index.md (hunk 24 above)
Comment (lscheske) [reply activities/communication/index.md:14]: They could also be part of the communications plan (I have left a comment there), what do you guys think?

File: activities/communication/communications-plan.md (hunk 30 above)
Comment (ElenaFdR) [reply activities/communication/communications-plan.md:66]: On a website, a call to action is [any action we invite the user to take](https://www.optimizely.com/nl/optimization-glossary/call-to-action/). A call to action shouldn't be persuasive; instead, it should help or prompt the user to accomplish a goal they have already formulated themselves (especially while they've read the preceding content).

File: activities/communication/blogging.md [hunk 34]
```diff
@@ -0,0 +1,98 @@
...
+* our community: interviews with and guest posts by community members
+* our position on important issues: why we’ve made the political, technological, organizational or design decisions we have
+
+Not all information published on the blog needs to support our marketing strategy - for example, an in-depth discussion about a technology choice we’ve made may not persuade potential Foundation members, but will be interesting to technologists in our community.
+
+To make sure we have the editorial freedom to publish about all of our work (no matter how niche), there’s no direct link to the blog on [publiccode.net](https://publiccode.net/). Instead, we may feature important blogposts. (We’ll build this functionality once we have important blogposts.)
+
+## Blogposts are ephemera
+
+Blogposts are different from [resources and guides published on About](../documentation/index.md), like a white paper we keep updated. By contrast, a blogpost is only true at that moment in time. We shouldn’t ever update a blogpost with more current information.
```
Comment (Ainali) [inline activities/communication/blogging.md:25]: The first sentence is a bit ambiguous. Is it the blogposts that are like white papers? Perhaps add a "who are" after the comma and before "like"?

File: activities/communication/communications-plan.md (hunk 17 above)
Comment (ElenaFdR) [reply activities/communication/communications-plan.md:50]: done.

File: activities/communication/blogging.md (hunk 16 above)
Comment (ElenaFdR) [reply activities/communication/blogging.md:40]: Is this specifically necessary for this activity? Surely this applies to everything we do?

File: activities/communication/blogging.md (hunk 21 above)
Comment (ElenaFdR) [reply activities/communication/blogging.md:21]: Can you raise this as an issue on the blog?

File: activities/communication/blogging.md (hunk 22 above)
Comment (ElenaFdR) [reply activities/communication/blogging.md:48]: Our blog isn't sophisticated enough to offer link configuration, but yes, otherwise this would be true.

File: activities/communication/blogging.md (hunk 23 above)
Comment (ElenaFdR) [reply activities/communication/blogging.md:73]: Done!

File: activities/communication/blogging.md (hunk 34 above)
Comment (ElenaFdR) [reply activities/communication/blogging.md:25]: I've clarified this now.

File: activities/communication/blogging.md (hunk 16 above)
Comment (Ainali) [reply activities/communication/blogging.md:40]: Yes, that should be implied. If not, we have a lot of more places to add it to.

File: activities/communication/communications-plan.md (hunk 27 above)
Comment (Ainali) [reply activities/communication/communications-plan.md:44]: Why not both?

File: activities/communication/communications-plan.md (hunk 33 above)
Comment (ElenaFdR) [reply activities/communication/communications-plan.md:119]: > [This comment has a soundtrack.](https://www.youtube.com/watch?v=WySzEXKUSZw)

I don't consider events a communications channel. The other channels are persistent, free to access, can be indexed by search engines and found well into the future, and all contain (interlinked or cumulative) content that we create for others to consume privately and asynchonously. Though we often communicate at events and host our own, and craft original content for them, most of the magic comes from being in the room with the other participants. 
It's best practice for us to share our event content publicly afterwards so it's accessible to people who couldn't make the event, but we do that  via the other communcations channels.

File: activities/communication/communications-plan.md (hunk 27 above)
Comment (ElenaFdR) [reply activities/communication/communications-plan.md:44]: It's a mix of both.

File: activities/communication/communications-plan.md (hunk 26 above)
Comment (ElenaFdR) [reply activities/communication/communications-plan.md:36]: Optimistically, all the people you list are future Foundation contributors, in the broadest sense of our community.

This list used to be [all of these people](https://about.publiccode.net/activities/value-and-impact/user-mapping/), but was cut to this to make it consistent with the very specific way we've defined 'user' (which hasn't been updated in value and impact mapping).

File: activities/communication/communications-plan.md (hunk 27 above)
Comment (ElenaFdR) [reply activities/communication/communications-plan.md:44]: Updated!

File: activities/communication/index.md (hunk 24 above)
Comment (ElenaFdR) [reply activities/communication/index.md:15]: I've added a link to events to the index page.

Comment (ElenaFdR) [reply activities/communication/index.md:15]: We currently don't have separate guidance for social media or webinars, so I can't add this now - but if we have such things in the future, then sure. I've also added a link to workshops.

**Review by bvhme — CHANGES_REQUESTED** [review]
Awesome, one technical thing, then it should be good to go.

**Review by bvhme — APPROVED** [review]
Great! Ready to merge.
